### DISCUSSION TITLE
Welcome to RandEmailAlias Discussions!

### DISCUSSION THREAD

**Discussion by JakeOrona** [JakeOrona/RandEmailAlias#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to provide feedback. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.

  To get started, comment below with any feedback or feature requests. Keep in mind this is a side project for me to practice python and learn how to manage a repo.

-BearBones 🐻

**Comment by JakeOrona** [comment]
I don't know if I like the current time format when using timestamp override. It is currently set to `HH-MM-SS.MICROSECONDS`.

This discission was made due to initially adding override functionality to `Feeling Lucky` button and output. After implementation it looked messy. The functionality was updated to override `Generate Alias Email` button.

![REAG-FeatureRequest-1](https://github.com/JakeOrona/RandEmailAlias/assets/25201464/8109325a-2ff0-45e3-b59d-05d040894019)

**Reply by JakeOrona** [reply]
✅  Updated timestamp format #15

**Comment by JakeOrona** [comment]
Add error logs/warnings? Currently no checks for valid input or messages to user.

**Reply by JakeOrona** [reply]
✅ basic error logging and email validation implemented.

**Comment by JakeOronaOE** [comment]
Format app frames vertically?

**Reply by JakeOrona** [reply]
✅ Updated to vertical format for smaller form factor #15

**Comment by JakeOronaOE** [comment]
[RAEG-MacOS Crash Report.txt](https://github.com/JakeOrona/RandEmailAlias/files/11455546/RAEG-MacOS.Crash.Report.txt)

First crash report

**Reply by versuslight** [reply]
baby's first crash report.

**Reply by JakeOrona** [reply]
I let my instance run all day without issue. Crash report doesn't say anything specific. Will continue to monitor.
